libpius/signer.py

```diff
@@ -645,6 +648,9 @@ def sign_uid(self, key, index, level):
     while True:
       line = gpg.stdout.readline()
       debug('Got %s' % line)
+      if 'MISSING_PASSPHRASE' in line:
```

Comment:
Ah. I wonder if that changed with agent v2 - back before gpg2 you got BAD whether you canceled or put in a bad passphrase.

In that case, let's change the messages, which is what I had wanted to do back in the day, but it wasn't possible:
For Bad:
```
ERROR: Agent reported the passphrase was incorrect.
```

For missing:
```
ERROR: Agent didn\'t provide passphrase to PGP.
```